api-server/src/server/boot/certificate.js

```diff
@@ -66,9 +66,11 @@ export function getFallbackFrontEndDate(completedChallenges, completedDate) {
   return latestCertDate ? latestCertDate : completedDate;
 }
 
+const slugs = Object.keys(slugCertTypeMap);
+
 function ifNoSuperBlock404(req, res, next) {
-  const { superBlock } = req.body;
-  if (superBlock && superBlocks.includes(superBlock)) {
+  const { slug } = req.body;
```

Comment:
`slug` is a bit generic.  It's a lot easier to maintain code when the name is a bit more specific, like, `certSlug`, `superBlockPath` or whatever is the description of how it's used.  Reason being, `slug` already appears in 30 files, so (unless they're all references to this) there's potential for confusion.

Reply:
Is this only used, and do we only ever plan to use this, to refer to the endpoint for the certs?

Reply:
Ironically, I'm not 100% sure because `superBlock` is used in so many places.  That said, I'm reasonably confident it only has this one use.